### PR TITLE
minor reformat to chyron readme. Added note about non-conformance of …

### PR DESCRIPTION
**Pull request by jarumihooi**
…the golds column headers as per conventions.

### CODE REVIEW

File: newshour-chyron/readme.md [hunk 1]
```diff
@@ -1 +1,107 @@
...
+Some decisions within the annotation process are described above.  
+
+> Precision Details
+
+**Boundaries** - This project was done with the annotation time as a subinterval. 
+This means the annotation will be fully within the time/frame where the chyron is occurring and is annotated as lasting up to while the chyron is still onscreen. 
+To repeat above, the choice was made to annotate the chyron as the first moment it stopped being transitory/fading-in, and the moment right before it begins to transition-out/fade-out. 
+(This is currently unvalidated.)  
+    > [!Note]  
+    > This is opposite the decision made for the Slates project.
```
Comment (keighrim) [inline newshour-chyron/readme.md:54]: these leading spaces might break the format...?

File: newshour-chyron/readme.md [hunk 2]
```diff
@@ -1 +1,107 @@
...
+### Preparation
+Videos are loaded into the VIA tool for annotation. 
+The annotator watches a video using the annotation tool, and labels chyrons in time.  
+
+### What to Annotate
+Usually, chyrons will fade in and out.  
+**START TIME** - When a chyron has fully appeared/solid and is no longer see through, pause the video and highlight the chyron via a bounding box. It's not too dissimilar from taking a screenshot.  
+**TEXT** - When the chyron is highlighted, double-click it and in the DESCRIPTION box, copy/write out what was said in the chyron exactly as it was displayed. When a chyron consists of more than one line, use the enter/return key (becomes \n in the gold output) to make a new line in the description.  
+**END TIME** - After that, submit how long the chyron was onscreen: annotating the end time as when it begins to become transparent.  
+_Time format: (sec.ten-thousandths of sec)_
```
Comment (keighrim) [inline newshour-chyron/readme.md:39]: wouldn't these be better with bullets?

File: newshour-chyron/readme.md [hunk 3]
```diff
@@ -1 +1,107 @@
...
+**END TIME** - After that, submit how long the chyron was onscreen: annotating the end time as when it begins to become transparent.  
+_Time format: (sec.ten-thousandths of sec)_   
+
+### How to Annotate It
+Please see VIA link for tool usage on step by step how to use the tool. 
+
+### Decisions, Differentiation, and Precision Level during Annotation
+Some decisions within the annotation process are described above.  
+
+> Precision Details
```
Comment (keighrim) [inline newshour-chyron/readme.md:47]: any particular reason to use quoting instead of sub-level heading...?

File: newshour-chyron/readme.md [hunk 4]
```diff
@@ -1 +1,107 @@
...
+    * Number of Batches - 1, (The batch is named batch2 likely as a name - there seems to be only one batch used.)
+    * Other Version Control Information - Unknown/1.0
+
+## Tool Installation: VIA tool
+_The following section is recalled from memory and may not be fully accurate._  
+[VIA3](https://www.robots.ox.ac.uk/~vgg/software/via/). This project used [this version](https://github.com/kelleyl/clams-via3/blob/master/app.py#L16) and the Video Annotator tool.  
+A video must be imported. Docker is likely used.  
+
+## Annotation Guidelines
+> This `readme.md` is the guideline for this project.
```
Comment (keighrim) [inline newshour-chyron/readme.md:28]: maybe add [!IMPORTANT] or [!NOTE]...?

File: newshour-chyron/readme.md [hunk 5]
```diff
@@ -1 +1,107 @@
...
+Please see VIA link for tool usage on step by step how to use the tool. 
+
+### Decisions, Differentiation, and Precision Level during Annotation
+Some decisions within the annotation process are described above.  
+
+> Precision Details
+
+**Boundaries** - This project was done with the annotation time as a subinterval. 
+This means the annotation will be fully within the time/frame where the chyron is occurring and is annotated as lasting up to while the chyron is still onscreen. 
+To repeat above, the choice was made to annotate the chyron as the first moment it stopped being transitory/fading-in, and the moment right before it begins to transition-out/fade-out.
```
Comment (keighrim) [inline newshour-chyron/readme.md:51]: Again, how about using bullets and sub-level bullets here?

**Review by keighrim — COMMENTED** [review]
Adding some suggestions for just cosmetics. Contents look very meticulous, great job 👍

**Review by jarumihooi — COMMENTED** [review]
(leaving comments only)

File: newshour-chyron/readme.md (hunk 3 above)
Comment (keighrim) [reply newshour-chyron/readme.md:47]: In my opinion, these "edge-case" handling should be a part of the guidelines (not a sibling).

File: newshour-chyron/readme.md (hunk 1 above)
Comment (keighrim) [reply newshour-chyron/readme.md:54]: Probably view a copy of the file in the actual commit. 

image

image

File: newshour-chyron/readme.md (hunk 3 above)
Comment (jarumihooi) [reply newshour-chyron/readme.md:47]: Just to check, you mean basically that it should be a ### subcategory within ## Annotation Guidelines. 

Yes, will use ####.

File: newshour-chyron/readme.md (hunk 1 above)
Comment (jarumihooi) [reply newshour-chyron/readme.md:54]: Ok, that preview shows it as broken. I'm removing the indent.

Comment (jarumihooi) [reply newshour-chyron/readme.md:54]: (removed indents)

**Review by keighrim — APPROVED** [review]

**Pull request merged**